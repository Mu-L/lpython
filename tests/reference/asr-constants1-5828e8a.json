{
    "basename": "asr-constants1-5828e8a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/constants1.py",
    "infile_hash": "5dca391c30a1477fb903e17c1d47dad3b9b816088384ba61ff372db1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-constants1-5828e8a.stderr",
    "stderr_hash": "e6a7dda4c89d36aeef81a85cc04ec6f87b5a0799a8df18641eb7a1f5",
    "returncode": 2
}