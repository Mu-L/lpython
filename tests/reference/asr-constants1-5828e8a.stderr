semantic error: The symbol 'bin' not found in the module 'lpython_builtin'
 --> tests/constants1.py:5:9
  |
5 |     b = bin(5)
  |         ^^^^^^ 
